doc/content/guides/interop-repository/_index.md

```diff
@@ -1,22 +1,45 @@
 ---
-title: "Interoperability repository"
+title: "Interoperability Repository"
 description: ""
 weight: 40
 ---
 
 ## Interoperability Repository
 
-In order to enable interoperability in the stack, create an interoperability configuration repository (e.g. a directory) with `config.yaml` at the root.
+In order to enable interoperability in the stack:
+1. Create an interoperability configuration repository (e.g. a directory) with `config.yaml` at the root
+2. Configure stack to use it via (environment variables given in parentheses):
```

Comment:
@htdvisser can you standardize the configuration instructions in sections?

I think we need to specify _only_ the yaml keys, and add a link to "how to configure the stack" which explains about yaml structure (config file), env vars and flags